content/en/docs/tasks/administer-cluster/securing-a-cluster.md

```diff
@@ -255,11 +256,14 @@ and may grant an attacker significant visibility into the state of your cluster.
 your backups using a well reviewed backup and encryption solution, and consider using full disk
 encryption where possible.
 
-Kubernetes supports [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/), a feature 
-introduced in 1.7, v1 beta since 1.13, and v2 alpha since 1.25. This will encrypt resources like `Secret` and `ConfigMap` in etcd, preventing
-parties that gain access to your etcd backups from viewing the content of those secrets. While
-this feature is currently beta, it offers an additional level of defense when backups
-are not encrypted or an attacker gains read access to etcd.
+As a beta feature, Kubernetes supports [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/).
```

Comment:
Only the v1 API is in beta, so should we call those out here? v1 - beta and v2 - alpha?

Reply:
Encryption at rest has been beta since Kubernetes v1.13, unless it graduated since then and the docs are stale.

The v1 config API is beta with an a v2 config API in alpha. That's not the same as the feature itself. We've had a bunch of features (eg the API for Pods) where the feature was stable, but a key API that enables them to actually work (eg CRI) was still alpha.

Reply:
Encryption at rest is GA (for secret box, aes, etc).

Encryption at rest for KMS v1 is beta.

Encryption at rest for KMS v2 is alpha.

Reply:
```suggestion
Kubernetes supports optional [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/)
for information in the Kubernetes API.
```
based on https://github.com/kubernetes/website/pull/38109#discussion_r1034199324